### DISCUSSION TITLE
Suggestion: Warehouse capacity info

### DISCUSSION THREAD

**Discussion by BloodyRain2k** [voxeltycoon/issues#957] [General]
Currently the only info warehouses show is their transfer speed.

It'd be nice however to actually see how many items can fit in them, given that there are currently 2 and probably more in the future (or after research).

After I was told that different wares have different capacities, it could be solved by for example adding a label to the info panel like (58) as the capacity, and then list the icons for all wares that fit 58 times into a warehouse.

This would be slightly less intuitive to look at at first, but much more space efficient than just a list of wares with their capacity.

Another idea I just had as I'm writing this, would be to just add a grid of all wares, sorted by capacity (biggest capacity first?) and either just add the capacity next to each, or have the ware name and capacity in a hover tooltip.

**Comment by IDragonfire** [comment]
Or just add the capacy range?

**Reply by BloodyRain2k** [reply]
How much good would that do if the warehouse can hold 58 coal and 5 cars and it'll just show "capacity: 5-58".
It'd be even more confusing in my opinion.

**Comment by IDragonfire** [comment]
PS.: Similar problems for the train info
![image](https://user-images.githubusercontent.com/1217999/164989887-819cb5da-6106-42a9-8ad9-73d822005e4b.png)

**Comment by kalmix88-hun** [comment]
I like the way the game calculates the space requirements of goods. It would be nice if this were also visible in the building menu. Currently, only the names of the warehouses (large, small, etc.) are displayed, and when I place the building, it shows how many units it can store.
